test/e2e/hca-helpers.js

```diff
@@ -166,214 +165,152 @@ const testValues = {
 }
 
-function completeSpouseInformation(client, data, onlyRequiredFields) {
-  selectDropdown(client, 'spouseBirthMonth', data.spouseDateOfBirth.month);
-  selectDropdown(client, 'spouseBirthDay', data.spouseDateOfBirth.day);
-  selectDropdown(client, 'marriageMonth', data.dateOfMarriage.month);
-  selectDropdown(client, 'marriageDay', data.dateOfMarriage.day);
+function completeSpouseInformation(client, data) {
   client
-    .clearValue('input[name="fname"]')
-    .setValue('input[name="fname"]', data.spouseFullName.first)
-    .clearValue('input[name="lname"]')
-    .setValue('input[name="lname"]', data.spouseFullName.last)
-    .clearValue('input[name="ssn"]')
-    .setValue('input[name="ssn"]', data.spouseSocialSecurityNumber)
-    .clearValue('input[name="spouseBirthYear"]')
-    .setValue('input[name="spouseBirthYear"]', data.spouseDateOfBirth.year)
-    .clearValue('input[name="marriageYear"]')
-    .setValue('input[name="marriageYear"]', data.dateOfMarriage.year)
+    .selectDropdown('spouseBirthMonth', data.spouseDateOfBirth.month)
+    .selectDropdown('spouseBirthDay', data.spouseDateOfBirth.day)
+    .selectDropdown('marriageMonth', data.dateOfMarriage.month)
+    .selectDropdown('marriageDay', data.dateOfMarriage.day)
+    .fill('input[name="fname"]', data.spouseFullName.first)
+    .fill('input[name="mname"]', 'Jacqueline')
+    .fill('input[name="lname"]', data.spouseFullName.last)
+    .selectDropdown('suffix', 'Sr.')
+    .fill('input[name="ssn"]', data.spouseSocialSecurityNumber)
+    .fill('input[name="spouseBirthYear"]', data.spouseDateOfBirth.year)
+    .fill('input[name="marriageYear"]', data.dateOfMarriage.year)
     .click('input[name="sameAddress-1"]');
   client.expect.element('input[name="address"]').to.be.visible.before(Timeouts.slow);
 
   client
-    .clearValue('input[name="address"]')
-    .setValue('input[name="address"]', data.spouseAddress.street)
-    .clearValue('input[name="city"]')
-    .setValue('input[name="city"]', data.spouseAddress.city);
-
-  selectDropdown(client, 'country', data.spouseAddress.country);
-  selectDropdown(client, 'state', data.spouseAddress.state);
-
-  client
-    .clearValue('input[name="zip"]')
-    .setValue('input[name="zip"]', data.spouseAddress.zipcode);
-
-  if (!onlyRequiredFields) {
-    selectDropdown(client, 'suffix', 'Sr.');
-    client
-      .setValue('input[name="mname"]', 'Jacqueline')
-      .click('input[name="sameAddress-1"]')
-      .click('input[name="cohabitedLastYear-0"]')
-      .click('input[name="provideSupportLastYear-0"]');
-  }
+    .fill('input[name="address"]', data.spouseAddress.street)
+    .fill('input[name="city"]', data.spouseAddress.city)
+    .selectDropdown('country', data.spouseAddress.country)
+    .selectDropdown('state', data.spouseAddress.state)
+    .fill('input[name="zip"]', data.spouseAddress.zipcode)
+    .click('input[name="cohabitedLastYear-1"]')
+    // Can't find this for some reason, but it doesn't fail...
```

Comment:
Oh, yeah I did find that. I just forgot to remove the comment. I couldn't find it because it was selecting 'Yes' when I wrote that.